src/main/resources/hudson/scm/SubversionSCM/ModuleLocation/help-cancelProcessOnExternalsFail.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Determines if the process should be cancelled when checkout/update svn:externals failed. Will work when "Ignore externals" box is not checked.
```

Comment:
Maybe it needs to be clarified to explicitly say that the default behavior is to fail the checkout

Reply:
Sorry for ignoring the code formatting while my IDE using different style, already correct it, and default behavior is clarified too.